packages/gantt/frontend/sirius-components-gantt/src/toolbar/Toolbar.tsx

```diff
@@ -67,11 +67,11 @@ export const Toolbar = ({
   const onFitToScreen = () => {
     const minTime = Math.min.apply(
       null,
-      tasks.filter((task) => Boolean(task.start)).map((task) => task.start.getTime())
+      tasks.filter((task) => Boolean(task.start)).map((task) => (task.start ? task.start.getTime() : 0))
```

Comment:
Your version is ok but we tend to write something like `task.start?.getTime() ?? 0`